tests/framework/validators/NumberValidatorTest.php

```diff
@@ -37,7 +69,13 @@ public function testValidateValueSimple()
         $this->assertTrue($val->validate(-20));
         $this->assertTrue($val->validate('20'));
         $this->assertTrue($val->validate(25.45));
+
+        $this->setPointDecimalLocale();
         $this->assertFalse($val->validate('25,45'));
+        $this->setCommaDecimalLocale();
+        $this->assertTrue($val->validate('25,45'));
```

Comment:
Values conversion is a separate issue. Here we're talking about user input only which is always done according to country tradition and is always considered valid for that particular country.